charts/oam-core-resources/values.yaml

```diff
@@ -3,9 +3,9 @@
 # Declare variables to be passed into your templates.
 
 replicaCount: 1
-useWebhook: true
+useWebhook: false
 image:
-  repository: oamdev/core-resource-controller:v0.3
+  repository: crossplane/addon-oam-kubernetes-local:v0.3
```

Comment:
Can you [retcon](https://en.wikipedia.org/wiki/Retroactive_continuity) this to v0.1 too, please.